kmir/k-src/mir-types.md

```diff
@@ -284,6 +285,10 @@ module MIR-TYPE-SYNTAX
                               | ByteStringLiteral
                               | Int
                               | Bool
+                              // | CapitalIdent // TEMPORARY
+                              // | Identifier // IDENTIFIER IS INCORRECT AS FIRST POSITION CANNOT BE UPPER CASE. NEED TO PRINT ACTUAL REPRESETATION
```

Comment:
s/REPRESETATION/REPRESENTATION/

I'm not sure exactly what you need here, but you could split `IdentifierToken`:
```
syntax FirstUpperIdentifierToken ::= r"[A-Z][_a-zA-Z0-9]*" 
syntax FirstOtherIdentifierToken ::= r"[_a-z][_a-zA-Z0-9]*"
```
then you could probably use something like this:
```
| FirstUpperIdentifierToken  [prefer]
| Identifier
```